engine/src/main/java/org/terasology/rendering/opengl/OpenGLUtil.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2015 MovingBlocks
+ * Copyright 2016 MovingBlocks
```

Comment:
I saw this file's name and I started thinking: should be Util or Utils. And then it occurred to me that we really shouldn't abbreviate: 

http://programmers.stackexchange.com/questions/72593/how-to-abbreviate-variable-names

Let's rename it simply to "Utilities". As it's in the OpenGL package we can drop the OpenGL prefix.